### ISSUE TITLE
Wallet Export

### ISSUE THREAD

**Issue by brandoniles** [barmstrong/bitcoin-android#6]
Had a request from a user to allow exporting the wallet.dat file.

**Comment by Rassah** [comment]
Seconded. At least have some way to back up the wallet manually. Backing it up to the cloud doesn't give access to it, and I can't find it on a non-rooted phone.

**Comment by Rassah** [comment]
I can confirm that reinstalling the app WILL erase your own wallet, and WILL overwrite whatever backup you may have in the cloud, so unless Google has an older backup of the wallet file and can give you access to it, your coins will be gone for ever as well.
So, PLEASE make the ability to back up a wallet to the SD card or computer a HIGH priority!

**Comment by barmstrong** [comment]
I agree this would be great.  The SD card can be read by any other application, so it's not a great place to keep backups unless we added encryption.  But maybe emailing yourself a backup?  Not sure.
